src/module.h

```diff
@@ -211,6 +211,7 @@ int TerminateModuleForkChild(int child_pid, int wait);
 ssize_t rdbSaveModulesAux(rio *rdb, int when);
 int moduleAllDatatypesHandleErrors(void);
 int moduleAllModulesHandleReplAsyncLoad(void);
+int moduleVerifyAllModulesAllowAtomicSlotMigrationOrReply(client *c);
```

Comment:
I would prefer it to be
```suggestion
int verifyAllModulesAllowAtomicSlotMigration(void);
```

Reply:
May I ask why? Are prefixes like these not useful?

Reply:
Yeah, I don't see any value.

Reply:
I like the prefixes, then I can fairly quickly find the corresponding file something is in. We normally omit the prefix for internal functions.

We do something more extreme inside aws, where everything in the same type of unit is like "amzUnit_functionName".

Reply:
I still think unit name in the method name seems redundant to me. If people actually benefit from this, I don't want to push this further.

AWS `amz` thing is to indicate internal code i.e. definitely helpful.

Reply:
I find it useful. I remember I forgot where `getNodeByQuery` is, so I had to use `git grep` to find it (or use a smart IDE, but I don't :laughing:).

We don't have to repeat the word "module" again in the same name though. How about this?

```suggestion
int moduleVerifyAllAllowAtomicSlotMigrationOrReply(client *c);
```

Reply:
My bad, I'm a IDE person, @murphyjacob4 feel free to resolve this conversation or take in Viktor's suggestion.

Reply:
Taking in Viktor's suggestion